app/src/main/java/org/mozilla/focus/settings/SettingsFragment.java

```diff
@@ -120,13 +120,26 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
     }
 
     @Override
-    public void onPrepareOptionsMenu(Menu menu) {
+    public void onPrepareOptionsMenu(final Menu menu) {
```

Comment:
\[PMD\] INFO: The method 'onPrepareOptionsMenu' has a Cyclomatic Complexity of 5\.